src/components/screens/profile/ProfileEditImage.tsx

```diff
@@ -0,0 +1,72 @@
+import { isWeb } from '@baca/constants'
+import { Box, Text, Button, Row, themeColors } from '@baca/design-system'
+import * as ImagePicker from 'expo-image-picker'
+import { t } from 'i18next'
+import React from 'react'
+import { Image, StyleSheet } from 'react-native'
+
+export const ProfileEditImage: React.FC = () => {
+  const [image, setImage] = React.useState<string | null>(null)
+
+  const pickImage = async () => {
```

Comment:
missing `useCallback` here

Reply:
Totally, I've to review some hooks to avoid rerenders and deliver high quality code, thank you so much Michał